packages/protocol/src/mdns/MdnsScanner.ts

```diff
@@ -1319,6 +1319,9 @@ export class MdnsScanner implements Scanner {
         } else if (result.T === 1) {
```

Comment:
I think this is redundant with the next check now

Reply:
removed